docs/source/index.rst

```diff
@@ -114,53 +120,25 @@ Demo Videos
 Related Papers
 ==============
 
-If you find our work useful in your research, please consider citing:
-
-Hardware & Experiments
-----------------------
-
-- Z. Wang, R. Spica and M. Schwager, “Game Theoretic Motion Planning for
-  Multi-Robot Racing,” In Proc. of the International Symposium on Distributed
-  Autonomous Robotics Systems (DARS 18), October, 2018.
-  `PDF <https://msl.stanford.edu/sites/default/files/wang-etal-dars18-mlt-rbt-racing.pdf>`_
-
-Trajectory Generation
----------------------
-
-- Z. Wang, S. Singh, M. Pavone and M. Schwager, “Cooperative Object Transport in
-  3D with Multiple Quadrotors using No Peer Communication,” In Proc. of the
-  International Conference on Robotics and Automation (ICRA), pp. 1064-1071,
-  2018.
-  `PDF <https://msl.stanford.edu/sites/default/files/wang.singh_.pavone.ea_.icra18.pdf>`_
-
-Differential Flatness
----------------------
-
-- D. Zhou, Z. Wang and M. Schwager, “Agile Coordination and Assistive Collision
-  Avoidance for Quadrotor Swarms Using Virtual Structures,” IEEE Transactions on
-  Robotics, vol. 34, no. 4, pp. 916-923, 2018.
-  `PDF <https://msl.stanford.edu/sites/default/files/zhou-etal-tro18-structure.pdf>`_
-
-- D. Zhou and M. Schwager, “Vector Field Following for Quadrotors using
-  Differential Flatness,” In Proc. of the International Conference on Robotics
-  and Automation (ICRA), pp. 6567-6572. 2014.
-  `PDF <https://msl.stanford.edu/sites/default/files/zhouschwagericra14quadvectorfield.pdf>`_
+Lab research using the MSL Quadrotors can be found in the
+`References section <references.html>`_.
 
 Contributing
 ============
 
-- Report bug or request feature by opening issues on Github
+- Report bugs or request features by opening issues on `our Github`_
 
 - Contribution is very welcome. Please fork the project and submit pull
   requests. New code will be reviewed before merging into the codebase.
 
 - Common functionality should be implemented in
```

Comment:
Making a controller is slightly more complicated that this, albeit pretty close. I don't think this add anything really for a user that just wants to get going and fly

Reply:
I think we need to go over the entire main page as a final review. So I'm going to leave these specific changes for later when we have a larger discussion about the main page's content and focus.